Source/Diagnostics/BoostedFrameDiagnostic.cpp

```diff
@@ -917,13 +1025,15 @@ writeMetaData ()
     BL_PROFILE("BoostedFrameDiagnostic::writeMetaData");
 
     if (ParallelDescriptor::IOProcessor()) {
-
-        if (!UtilCreateDirectory(WarpX::lab_data_directory, 0755))
-            CreateDirectoryFailed(WarpX::lab_data_directory);
+        //const std::string fullpath = WarpX::lab_data_directory + "/snapshots";
```

Comment:
Could we get rid of this line?

Reply:
Done. replaced  
const std::string fullpath = WarpX::lab_data_directory ; with 
const std::string fullpath = WarpX::lab_data_directory + "/snapshots";